### PR TITLE
Full Site

### PR DESCRIPTION
**Pull request by Othniel2471**
### In this branch i :

- Installed and used React routing.
- Created a navbar component to house the navigation for the website.
- Created separate pages for each page.
- Styled each page using react styling methods.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,14 @@
 
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <br/>
+
+  <h3><b>Math Magician</b></h3>
+
+</div>
```
Comment (BenMukebo) [inline README.md:10]: - [x] When I click the link to the [issues page](https://github.com/Othniel2471/math-magician/blob/issues) link that you added, I get this 404 `error`. I would recommend adding the issue link in the contributing sections: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).
```

File: src/styles/calculatorPage.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
+.calcs {
+  display: flex;
+  flex-direction: row;
```
Comment (BenMukebo) [inline src/styles/calculatorPage.css]: - [ ] This is great using flexbox to layout your content, congrats 👏 , But we can still improve this code to No repeating of the same rule by using DRY (don't repeat yourself) but why? well, I realized that you have used the property `display: flex` at multiple places, please let's keep the code clean by extracting the property into a separate class that can be reused at multiple places, see the example below: 

- CSS file
```css
.d-flex {
  display: flex;
}

.flex-center {
  flex-direction: column;
  align-items: center;
}
```
- JS file
```js
<div className="d-flex flex-center">{your content}</div>
```

And remember you can do this for all the other properties that seem to be repeated multiple times, With this, I promise you'll be coding like a pro 😉

**Review by BenMukebo — CHANGES_REQUESTED** [review]
You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻
- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
- Descriptive README file.  ✔️ 
- Your Design looks great. 👍🏻 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL ] Use descriptive commit messages.  The commit message should be written in the imperative tense and capitalized. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8). Example:
  - :x: Wrong: added Descriptive README
  - :heavy_check_mark: Right: Add descriptive README

- [OPTIONAL] Kindly avoid using the slah ( / ) sign when naming your feature branch, this creates a new apps directory 👉🏻 [../feature/full-site]() 


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Othniel2471, 👋 👋  

Your project is complete! Great job for following the project requirement🥇 and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 Congratulations!  🎉💯🌟 




## Good Points 👍

- Descriptive Pull Request title & summary. ✔️
- Website consisting of 3 pages: `Home`, `Calculator`, and `Quote` ✔️
- [React Router V6](https://reactrouter.com/) Used for Proper Navigation ✔️
- Good use of Git flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**